using-timescaledb/continuous-aggregates.md

```diff
@@ -16,145 +16,201 @@ every refresh. Only new and/or invalidated data will be calculated.  Since this
-materialization will generally be 2 hours behind the current time.
-Tuning the `refresh_lag` parameter lower will mean that the
-aggregates will follow inserts more closely, but can cause some write
-amplification which may degrade insert performance. Setting `refresh_lag` to
-`-<bucket_width>` will keep the continuous aggregate up-to-date.
-
-The `refresh_interval` controls how frequently materialization jobs will be
-launched. Setting a shorter interval will mean materializations happen more
-"eagerly" but can consume background worker resources while it is running.
-
-The `timescaledb.max_interval_per_job` parameter can also be useful, especially
-when a continuous aggregate is created on a hypertable that already contains
-data and the materialization job needs to catch up. It determines the maximum
-amount of data processed in a single job. If there is still more work to do to
-catch up to where we need to be based on the `refresh_lag` setting, another job
-will be spawned immediately following completion. This means that the part of the
-data processed by previous jobs will be available even before the
-materialization is fully caught up.
+### Advanced Usage
+
+When querying the continuous aggregate view (for example,
+`conditions_summary_daily`) you want to get a complete view of the
+data including both the partially aggregated data the data in the
+underlying table (for example, `conditions`). For this reason, the
+view will get one part of the result from the partially aggregated
+data and one part from the underlying table.
+
+If you, however, want to just get the data from the materialized
+hypertable, and not include data from the underlying table, you can
+set the option `timescaledb.materialized_only` to `true` using `ALTER
+VIEW`:
+
+```sql
+ALTER VIEW conditions_summary_hourly SET (
+    timescaledb.materialized_only = true
+);
+```
+
+When designing continuous aggregates, we wanted to avoid write
+amplification since this drastically affect insert rate for
+write-heavy time-series workloads. As a result, data is aggregated
+into a materialized hypertable (one for each continuous aggregate
+view) containing partial aggregates trailing the underlying hypertable
+(the table `conditions` in the example above).
+
+>:TIP: You can find the name of the materialized hypertable in the
+>`timescaledb_information.continuous_aggregates`.
+>
+> ```sql
+> postgres=# SELECT view_name, materialization_hypertable
+> postgres-#   FROM timescaledb_information.continuous_aggregates;
+>          view_name         |            materialization_hypertable             
+> ---------------------------+---------------------------------------------------
+>  conditions_summary_hourly | _timescaledb_internal._materialized_hypertable_30
+>  conditions_summary_daily  | _timescaledb_internal._materialized_hypertable_31
+> (2 rows)
+> ```
+
+Each continuous aggregate use the *bucket width* of the `time_bucket`
```

Comment:
I guess this section is what it is now and we'll keep it. But I don't understand what purpose it serves and how it helps users. It's basically describing our internal implementation and if this level of understanding is needed to use continuous aggregates properly, then we've failed.

We generally don't  go into this depth about other features, for good reasons.

Reply:
```suggestion
Each continuous aggregate uses the *bucket width* of the `time_bucket`
```